docs/install/installing-istio.md

```diff
@@ -29,235 +29,180 @@ instructions. If you're familiar with Istio and know what kind of installation
 you want, read through the options and choose the installation that suits your
 needs.
 
-You can easily customize your Istio installation with `helm`. The below sections
+You can easily customize your Istio installation with `istioctl`. The below sections
 cover a few useful Istio configurations and their benefits.
 
 ### Choosing an Istio installation
 
 You can install Istio with or without a service mesh:
 
-- _automatic sidecar injection_: Enables the Istio service mesh by
-  [automatically injecting the Istio sidecars][1]. The sidecars are injected
-  into each pod of your cluster as they are created.
-
-- _manual sidecar injection_: Provides your Knative installation with traffic
-  routing and ingress, without the Istio service mesh. You do have the option of
-  later enabling the service mesh if you [manually inject the Istio
-  sidecars][2].
-
-If you are just getting started with Knative, we recommend installing Istio
-without automatic sidecar injection.
-
-### Downloading Istio and installing CRDs
-
-1. Enter the following commands to download Istio:
-
-   ```shell
-   # Download and unpack Istio
-   export ISTIO_VERSION=1.4.6
-   curl -L https://git.io/getLatestIstio | sh -
-   cd istio-${ISTIO_VERSION}
-   ```
-
-1. Enter the following command to install the Istio CRDs first:
-
-   ```shell
-   for i in install/kubernetes/helm/istio-init/files/crd*yaml; do kubectl apply -f $i; done
-   ```
-
-   Wait a few seconds for the CRDs to be committed in the Kubernetes API-server,
-   then continue with these instructions.
-
-1. Create `istio-system` namespace
-
-   ```shell
-   cat <<EOF | kubectl apply -f -
-   apiVersion: v1
-   kind: Namespace
-   metadata:
-     name: istio-system
-     labels:
-       istio-injection: disabled
-   EOF
-   ```
-
-1. Finish the install by applying your desired Istio configuration:
-   - [Installing Istio without sidecar injection](#installing-istio-without-sidecar-injection)(Recommended
+- [Installing Istio without sidecar injection](#installing-istio-without-sidecar-injection)(Recommended
      default installation)
-   - [Installing Istio with sidecar injection](#installing-istio-with-sidecar-injection)
-   - [Installing Istio with SDS to secure the ingress gateway](#installing-istio-with-sds-to-secure-the-ingress-gateway)
 
-#### Installing Istio without sidecar injection
+- [Installing Istio with sidecar injection](#installing-istio-with-sidecar-injection)
 
 If you want to get up and running with Knative quickly, we recommend installing
 Istio without automatic sidecar injection. This install is also recommended for
 users who don't need the Istio service mesh, or who want to enable the service
-mesh by [manually injecting the Istio sidecars][2].
+mesh by [manually injecting the Istio sidecars][1].
 
 Enter the following command to install Istio:
```

Comment:
This seems really complicated. I was able to get both `default` and `default with SDS` with the following two commands on Istio 1.4.9 (tested yesterday when I rebuilt my cluster):

```shell
istioctl manifest apply
```

```shell
istioctl manifest apply --set values.gateways.istio-ingressgateway.sds.enabled=true
```

I think those both install the sidecar; `--set autoInjection.enabled=false` I think disables the sidecar injection.

(I'm just hoping to have a minimal command line so it's not so intimidating.)

Reply:
Thank you @evankanderson
The reason of this complicated definition is that `cluster-local-gateway`. We cannot setup `cluster-local-gateway` with command line and so need to use this IstioOperator definition.

The installation by `istioctl manifest apply` works for external access but we cannot use `.svc.cluster.local` for ksvc.

Reply:
Ah, thanks! I wasn't testing eventing, only external services, so I missed this. It might be worth explaining that the `cluster-local-gateway` is needed for serving+eventing use.